src/Codeception/Configuration.php

```diff
@@ -500,6 +500,58 @@ public static function projectDir()
     }
 
     /**
+     * Returns path to a given directory relative to your project.
+     * @param string $path
+     * @return string
+     */
+    public static function getRelativeDir($path)
+    {
+        $relPath = $path;
+        if (substr($path, 0, strlen(self::projectDir())) == self::projectDir()) {
+            $relPath = substr($path, strlen(self::projectDir()));
+        } else {
+            $pathPrefix = self::projectDir();
+            $sameAbsoluteness = true;
+            $rootPathIndex = 0;
+            if (DIRECTORY_SEPARATOR == '\\') {
+                $hasDriveSpecifier = preg_match('/^[A-Za-z]:/', $pathPrefix);
+                $sameAbsoluteness = (preg_match('/^[A-Za-z]:/', $pathPrefix) == preg_match('/^[A-Za-z]:/', $path));
+                $rootPathIndex = $hasDriveSpecifier ? 2 : 0;
+                if ($sameAbsoluteness && $hasDriveSpecifier && preg_match('/^[A-Za-z]:/', $pathPrefix)) {
+                    $sameAbsoluteness = ($pathPrefix[0] == $path[0]);
+                }
+            }
+            if ($sameAbsoluteness) {
```

Comment:
I mentioned when I submitted the PR, that the code was pretty rough. Basically you can only make a relative path of one path against the other if they both have the same "absoluteness":

* On *nix this is easy:
    * A path is absolute if it starts with a slash (a Boolean)
    * Two paths have the same absoluteness if both or neither start with a slash
* On Windows this is less clear:
    * a path can be relative to the current or a specified device
    * it can also be absolute or relative to the CWD of that device
    * If a drive is specified, they both have to specify the same drive
    * If no drive is specified, they both need to *not* specify a drive
    * whether or not they specify a drive letter the first non-drive letter character (after the `:`) need to both or neither start with a backslash.

If the paths don't have the same absoluteness, you can't make a relative path because they're *not* relative to one another... 

I need to clarify the code to capture that more clearly, but does that clarify the logic to you at least?